libraries/tools/kotlin-gradle-plugin-idea/src/main/kotlin/org/jetbrains/kotlin/gradle/kpm/idea/IdeaKotlinProjectModel.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2010-2022 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.gradle.kpm.idea
+
+import java.io.File
+import java.io.Serializable
+
+interface IdeaKotlinProjectModel : Serializable {
```

Comment:
Does it make sense to add KGP version, so there is graceful handling in case of incompatibility?

Reply:
Yes, this makes a lot of sense!
I added an additional field `gradlePluginVersion: String`  to the model 
✅